### PR TITLE
DSAA wrapped 2023 blog post

### PR DESCRIPTION
**Pull request by meggie-d**
# Summary

Provide a summary of your blog post here.

# Checklist (Writer)

- [x] I can successfully knit the blog post Rmd file.

- [x] Figures, graphs, tables render properly in the blog post Rmd file.

- [x] Abbreviations are expanded/explained.

- [x] The blog post Rmd file header contains the post title, post description, author information, and categories.

- [x] The blog post content (Rmd file, figures, tables, etc.) does not contain any PHI.

# Checklist (AA Blog Team Reviewers)

- [ ] I have updated the blog's draft status from 'TRUE' to 'FALSE'

- [ ] I have updated the date on the blog post to reflect the release date

- [ ] I have renamed the blog post directory so the date matches the release date

- [ ] I have re-rendered the blog post and all required files are present

### CODE REVIEW

File: _posts/2023-12-20-dsaa-wrapped-2023/dsaa_wrapped_2023.jpg
Comment (chloepouprom) [inline _posts/2023-12-20-dsaa-wrapped-2023/dsaa_wrapped_2023.jpg:1]: This is awesome!

File: _posts/2023-12-20-dsaa-wrapped-2023/dsaa-wrapped-2023.Rmd [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- [Clean code in Python](https://github.com/zedr/clean-code-python)  
+- and more!
+
+This year, our Advanced Analytics (AA) sub-team had **more than 10** journal club sessions that dived into the worlds of Natural Language Processing (NLP), performance optimization, and project evaluation. The AA team also had the chance to take their learning outside of the office, by going on a field trip to the Thomas Fisher Rare Book Library at U of T, which had an exhibit of [Data Visualization Throughout History](https://fisher.library.utoronto.ca/exhibition/emerging-patterns-data-visualization-throughout-history).
+
+We were also fortunate to have Posit host a number of workshops for us, covering the topics of [pointblank](https://github.com/rstudio/pointblank) and [MLOps with vetiver](https://juliasilge.github.io/mlops-with-vetiver/#/title-slide).
+
+# Sharing ideas
+
+This year, we didn't just learn new things, our team members also had opportunities to exchange ideas, and share insights and experiences through talks and conferences from MIQI to the Renal Insight User Group.
```
Comment (chloepouprom) [inline _posts/2023-12-20-dsaa-wrapped-2023/dsaa-wrapped-2023.Rmd:69]: Would be good to include links to these conferences

File: _posts/2023-12-20-dsaa-wrapped-2023/dsaa-wrapped-2023.Rmd [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+We also hope to continue to raise our count of published blog posts! This past year, we've had team members share **3 blog posts** about [developing and deploying AI solutions in the hospital](https://lks-chart.github.io/blog/posts/2023-01-27-with-a-little-help-from-my-friends/), [creating reproducible environments for building medical imaging machine learning models with Nix](https://lks-chart.github.io/blog/posts/2023-02-28-medical-imaging-with-nix/), and [discrete event simulation in healthcare](https://lks-chart.github.io/blog/posts/2023-03-17-introduction-to-discrete-event-simulations-in-healthcare/).
+
+# The Party Planning Pandas and beyond! 🎉
+
+Let's conclude our DSAA Wrapped by recounting some of our memorable team moments from this past year, organized by our social committee: The Party Planning Pandas 🐼.
+
+- Nerf Gun Challenge: With our in-office stash of nerf guns, during this challenge, our team learned that precision and luck go hand in hand.  
+- Valentine's Day & Zoom trivia: This year, our data science team embraced AI art generators to craft charming (and maybe a little scary) Valentine's Day cards as well as adorable baby animal versions of our team.
```
Comment (chloepouprom) [inline _posts/2023-12-20-dsaa-wrapped-2023/dsaa-wrapped-2023.Rmd:78]: Do we have any of these saved? Would be cool to share (if they're not too scary!!)

**Review by chloepouprom — APPROVED** [review]

**Review by derekbeaton — APPROVED** [review]
It's great, no comments! Approved!

**Comment by derekbeaton** [comment]
@meggie-d feel free to merge and post!

**Pull request merged**